docs/changelog.rst

```diff
@@ -2,6 +2,7 @@
 Changelog
 =========
 
+* :bug:`-` Now querying ens names for an empty list of addresses won't be causing an error.
```

Comment:
Should also add a changelog for this very fix :)

It's a nice bug that you are fixing. In fact this entire PR should have been against bugfixes.

Also put all bugs in the bottom and all features on top.